db/data/20220921084032_fix_apply_draft_training_routes.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class FixApplyDraftTrainingRoutes < ActiveRecord::Migration[6.1]
+  def up
+    Trainee.draft.where(record_source: RecordSources::APPLY).where.not(course_uuid: nil).find_each do |trainee|
+      if trainee.published_course && trainee.training_route != trainee.published_course.route
+        trainee.without_auditing do
+          trainee.progress.funding = false
```

Comment:
i'm wondering whether we can exclude blatting the funding for trainees on course subjects that don't attract funding at all? or something like that 🤔

Not sure what the numbers are like, so maybe it's not worth it.

Reply:
Only about 120 trainees will be affected.